projects/elife.yaml

```diff
@@ -1972,6 +1972,15 @@ elife-xpub:
             s3:
                 "{instance}-elife-xpub":
                      deletion-policy: retain
+            rds:
+                multi-az: true
+                engine: postgres # or 'MySQL'
+                version: '10.4'
+                type: db.t2.small
+                storage: 5 # GB
+                storage-type: gp2
+                backup-retention: 28 # days
+                deletion-policy: Snapshot
```

Comment:
What does the deletion policy do?

Reply:
In case the database is deleted, it takes a snapshot before doing so. Resources in `prod` are stopped from being deleted anyway by some IAM policies we have.